platformio.ini

```diff
@@ -902,65 +907,69 @@ build_flags   = ${common_stm32.build_flags}
 platform          = ${common_stm32f1.platform}
 extends           = common_stm32f1
 board             = genericSTM32F103VE
+platform_packages = ${common_stm32f1.platform_packages}
```

Comment:
You can delete these if they are not adding any other dependencies. They will automatically inherit from the parent due to the `extends` line.